package.json

```diff
@@ -39,7 +40,7 @@
     "standard-version": "^5.0.2"
   },
   "peerDependencies": {
-    "nuxt": "<1.0.0 || >1.2.1"
+    "nuxt": ">2.0.0"
```

Comment:
As we have nuxt-edge do you think peerDependencies is needed?

Reply:
ye, right, removed